_includes/general/navbtn.html

```diff
@@ -1,8 +1,8 @@
 <svg role="img" width="20" height="20" aria-labelledby="ntitle" id="navbtn" class="hide-on-dt">
   <title id="ntitle">{{ site.data.general.nav.btnOpenNavigation }}</title>
   <g>
-    <rect fill="currentcolor" id="r1" width="100%" height="3" x="0" y="0" />
-    <rect fill="currentcolor" id="r2" width="100%" height="3" x="0" y="8" />
-    <rect fill="currentcolor" id="r3" width="100%" height="3" x="0" y="16" />
+    <rect fill="currentcolor" id="r1" width="100%" height="3" x="0" y="0">
```

Comment:
Firefox mobile dev mode doesn't seem to like this change. The last two hamburger rectangles disappear.  SVG's need self closing tags.

Reply:
I think I fixed it now.